Makefile

```diff
@@ -27,7 +27,7 @@ check-xml:
 	python -m subunit.run testrepository.tests.test_suite | subunit2junitxml -o test.xml -f | subunit2pyunit
 
 release:
-	python3 -m build --sdist --wheel --outdir dist
+	hatchling build
```

Comment:
My projects all use `hatch build`, is this doing anything different?

Reply:
At least on Debian "hatchling" is the only of the two commands that is packaged, and it works for me.

Reply:
Ok, it's just that it's not in the docs for building (https://hatch.pypa.io/latest/build/) or CLI reference (https://hatch.pypa.io/latest/cli/reference/#hatch-build) so you're possibly using an undocumented feature that could disappear.

`hatch` is the primary CLI so that's a bit weird it's not packaged.

Reply:
There's a long discussion here about the relation between the two here: https://github.com/pypa/hatch/issues/120

It's a bit long so I haven't bothered reading in detail, though it seems like we'll probably eventually end up with one.

I'll merge this since it's only relevant for the release manager, but we can always change it.